extensions/positron-r/src/session.ts

```diff
@@ -697,6 +697,8 @@ export function createJupyterKernelSpec(
 		// Workaround for
 		// https://github.com/posit-dev/positron/issues/1619#issuecomment-1971552522
 		env['LD_LIBRARY_PATH'] = rHomePath + '/lib';
+	} else if (process.platform === 'darwin') {
+		env['DYLD_LIBRARY_PATH'] = rHomePath + '/lib';
```

Comment:
```suggestion
	} else if (process.platform === 'darwin') {
	        // Workaround for
	        // https://github.com/posit-dev/positron/issues/3732
		env['DYLD_LIBRARY_PATH'] = rHomePath + '/lib';
```

To match the linux branch and give us something to look back on